src/Core/View.js

```diff
@@ -224,6 +229,9 @@ View.prototype.addLayer = function addLayer(layer, parentLayer) {
 View.prototype.notifyChange = function notifyChange(needsRedraw, changeSource) {
     this._changeSources.add(changeSource);
     this.mainLoop.scheduleViewUpdate(this, needsRedraw);
+    if (needsRedraw) {
+        this.dispatchEvent({ type: VIEW_EVENTS.UPDATED });
```

Comment:
It's interesting to know when the viewer is updated to synchronize with other applications
See examples with `layersColorVisible.html `, that I modified to couple the viewer and the menu